api_v3/services/LiveStreamService.php

```diff
@@ -538,12 +535,9 @@ public function regenerateStreamTokenAction($entryId)
 		$password = sha1(md5(uniqid(rand(), true)));
 		$password = substr($password, rand(0, strlen($password) - 8), 8);
 		$liveEntry->setStreamPassword($password);
-		
-		$broadcastUrlManager = kBroadcastUrlManager::getInstance($liveEntry->getPartnerId());
```

Comment:
Since in liveEntry the url is returned as is if exists in customData we cannot remove these calls.
We need to regenerate the token and save it on the entry object.